easybuild/easyconfigs/r/R/R-3.6.2-foss-2019b.eb

```diff
@@ -0,0 +1,2305 @@
+        'checksums': ['d6934dca8f835d8173e3fb9fd4d5e2740c8c04348dd2bcc57df1b711facb46bc'],
+    }),
+    ('clisymbols', '1.2.0', {
+        'checksums': ['0649f2ce39541820daee3ed408d765eddf83db5db639b493561f4e5fbf88efe0'],
+    }),
+    ('ini', '0.3.1', {
+        'checksums': ['7b191a54019c8c52d6c2211c14878c95564154ec4865f57007953742868cd813'],
+    }),
+    ('gh', '1.0.1', {
+        'checksums': ['f3c02b16637ae390c3599265852d94b3de3ef585818b260d00e7812595b391d2'],
+    }),
+    ('usethis', '1.5.1', {
+        'checksums': ['9e3920a04b0df82adf59eef2c1b2b4d835c4a757a51b3c163b8fc619172f561d'],
+    }),
+    ('DT', '0.10', {
+        'checksums': ['3cc6dfc9697b52aef21d30dcfd355831c6216edcc6dd6bfb9a106cce6ab0906f'],
+    }),
+    ('rex', '1.1.2', {
+        'checksums': ['bd3c74ceaf335336f5dd04314d0a791f6311e421a2158f321f5aab275f539a2a'],
+    }),
+    ('covr', '3.4.0', {
+        'checksums': ['e57d9c656b4ab50a7f9557dee422d8777efaf08ecde0fd8bfa5897ef4d9e845c'],
+    }),
+    ('devtools', '2.2.1', {
+        'checksums': ['2e988fb56b068ba958ea471224d2462427868ce7706e157a90c0ce0e13294e44'],
+    }),
+    ('Rook', '1.1-1', {
+        'checksums': ['00f4ecfa4c5c57018acbb749080c07154549a6ecaa8d4130dd9de79427504903'],
+    }),
+    ('Cairo', '1.5-10', {
+        'checksums': ['7837f0c384cd49bb3342cb39a916d7a80b02fffbf123913a58014e597f69b5d5'],
+    }),
+    ('RMTstat', '0.3', {
+        'checksums': ['81eb4c5434d04cb66c749a434c33ceb1c07d92ba79765d4e9233c13a092ec2da'],
+    }),
+    ('Lmoments', '1.3-1', {
+        'checksums': ['7c9d489a08f93fa5877e2f233ab9732e0d1b2761596b3f6ac91f2295e41a865d'],
+    }),
+    ('distillery', '1.0-6', {
+        'checksums': ['4910e2952f767c1062d7cbe648c90a97009e2b3da316c6b33f6d022cd38b23d6'],
+    }),
+    ('extRemes', '2.0-11', {
+        'checksums': ['75fbdeef677c81cf5661b8df3df4090c55f53e9bb96bb138b498eb0fbbf5af42'],
+    }),
+    ('pixmap', '0.4-11', {
+        'checksums': ['6fa010749a59cdf56aad9f81271473b7d55697036203f2cd5d81372bcded7412'],
+    }),
+    ('tkrplot', '0.0-24', {
+        'checksums': ['2873630a37d7ae1e09a5803d9a89ca0494edd83526c7b1860d9246543722f311'],
+    }),
+    ('misc3d', '0.8-4', {
+        'checksums': ['75de3d2237f67f9e58a36e80a6bbf7e796d43eb46789f2dd1311270007bf5f62'],
+    }),
+    ('multicool', '0.1-11', {
+        'checksums': ['1c907e64af2ac39facdf431a5691e69649f64af1f50e198ae39da5bf30026476'],
+    }),
+    ('plot3D', '1.1.1', {
+        'checksums': ['f6fe4a001387132626fc553ed1d5720d448b8064eb5a6917458a798e1d381632'],
+    }),
+    ('plot3Drgl', '1.0.1', {
+        'checksums': ['466d428d25c066c9c96d892f24da930513d42b1bdf76d3b53628c3ba13c3e48a'],
+    }),
+    ('OceanView', '1.0.4', {
+        'checksums': ['e67f6f376737e1e5cf22fdfe2769a8f674e90c886b0e43942e97d9a3e6924f1c'],
+    }),
+    ('ks', '1.11.6', {
+        'checksums': ['d8f1ccb99281eed092b8f950cfbf422330e88f2a0ce1ef6c9dbd7d8b9d648c41'],
+    }),
+    ('logcondens', '2.1.5', {
+        'checksums': ['72e
```

Comment:
@edmondac This depends on `Biobase`, which is a Bioconductor package (so it belongs in that `R-bundle-Bioconductor` easyconfig instead)...

Reply:
How odd, it's been in the main R build since 2016a.

Are you sure? I can't see any mention of Bio here: https://cran.r-project.org/web/packages/logcondens/index.html

Reply:
Ah - "ERROR: dependency Biobase is not available for package multtest"

Reply:
Yeah, this was supposed to go where `multtest` is, sorry about the confusion. :)